src/main/java/uk/gov/hmcts/reform/sscs/service/VenueService.java

```diff
@@ -24,13 +24,17 @@ public String getEpimsIdForVenue(String processingVenue) {
         String venueId = String.valueOf(airLookupService.getLookupVenueIdByAirVenueName()
             .get(processingVenue));
 
+        return getEpimsIdForVenueId(venueId);
+    }
+
+    public String getEpimsIdForVenueId(String venueId) {
         VenueDetails venueDetails = venueDataLoader.getVenueDetailsMap()
             .get(String.valueOf(venueId));
 
         return Optional.ofNullable(venueDetails)
             .map(VenueDetails::getEpimsId)
             .orElseThrow(() -> {
-                throw new IllegalStateException("Unable to find epims ID for processing venue: " + processingVenue);
+                throw new IllegalStateException("Unable to find epims ID for venue ID: " + venueId);
             });
```

Comment:
I personally think this is a overengineered way to do
```
if (isNull(venueDetails)) {
   throw new IllegalStateException("Unable to find epims ID for venue ID: " + venueId);
}
return venueDetails
```

Reply:
These changes are related to a different ticket 10293. I will remove them from this PR and @nickhill111 will implement them on his own branch.